packages/drpc/src/messages/DrpcRequest.ts

```diff
@@ -0,0 +1,73 @@
+import type { ValidationArguments, ValidationOptions } from 'class-validator'
+
+import { IsValidMessageType, parseMessageType, AgentMessage } from '@credo-ts/core'
+import { Expose } from 'class-transformer'
+import { ValidateBy, ValidationError, buildMessage } from 'class-validator'
+
+export function IsValidDrpcRequest(validationOptions?: ValidationOptions): PropertyDecorator {
+  return function (target: any, propertyKey: string | symbol) {
```

Comment:
Is this a validator requirement that `target` is `any` or can we use `unknown` as well?

Reply:
Would also be good to move the validation functions to another file and just keep this the request class.

Reply:
it has to be any or Object

Reply:
I think it expects the target to be type `Object` but having `Object` as a type creates a linting error